lib/order.rb

```diff
@@ -0,0 +1,34 @@
+class Order
+
+  attr_reader :basket, :current_total, :dish_list
+
+  def initialize(dish_list)
+    @dish_list = dish_list
+    @basket = Hash.new(0)
```

Comment:
Is it necessary to use a Hash here? I get that you can represent quantity nicely like this, but it feels simpler to use an array of dishes.

It might simplify the add to order and total price bits too.